README.md

```diff
@@ -1 +1,13 @@
-# hw01
\ No newline at end of file
+# hw01
+## Introduction
```

Comment:
You should leave a blank line after headers. It's not an issue here, but once you begin working with code chunks in RMarkdown files later, this practice has the potential to save you some formatting troubles.